File: 05/elixir/main.exs
{f, _} = Integer.parse IO.gets("What is the first number?\n")
{s, _} = Integer.parse IO.gets("What is the second number?\n")
IO.puts "#{f} + #{s} = #{f + s}"
IO.puts "#{f} - #{s} = #{f - s}"
IO.puts "#{f} * #{s} = #{f * s}"
IO.puts "#{f} / #{s} = #{f / s}"

File: 05/main.clj
(println "What is the first number?")
(let [f (Integer. (re-find #"\d+" (read-line)))]
  (println "What is the second number?")
  (let [s (Integer. (re-find  #"\d+" (read-line)))]
    (println (str f " + " s " = " (+ f s)))
    (println (str f " - " s " = " (- f s)))
    (println (str f " * " s " = " (* f s)))
    (println (str f " / " s " = " (/ f s)))))
